package.json

```diff
@@ -6,9 +6,9 @@
   "bugs": "https://github.com/adobe/aio-cli-plugin-app/issues",
   "dependencies": {
     "@adobe/aio-app-scripts": "^2.0.0",
-    "@adobe/aio-lib-core-config": ">=1.2.8",
+    "@adobe/aio-lib-core-config": "2.0.0",
```

Comment:
should be caret? we want to pick up any path or minor releases as well

Reply:
+1 my bad